zos_basics/factgathering/gather-facts.yml

```diff
@@ -0,0 +1,82 @@
+###############################################################################
+# © Copyright IBM Corporation 2022
+###############################################################################
+# This sample playbook demonstrates the zos fact gathering module included in
+# the Red Hat Ansible Certified  Content for IBM Z core collection.
+#
+# Usage:
+#  ansible-playbook -i <inventory> <playbook>
+#
+# Example:
+#  ansible-playbook -i inventories gather-facts.yml
+#  ansible-playbook -i inventories gather-facts.yml -v
+#
+# Requirements:
+#   IBM z/OS core collection 1.4.0 or later. # TODO update this version!
+#
+###############################################################################
+
+- name: z/OS Gather Facts Sample Playbook
+  collections:
+    - "ibm.ibm_zos_core"
+  hosts: zos_host
+
+  environment: "{{ environment_vars }}"
+
+  gather_facts: false
+
+  tasks:
+
+    # ##########################################################################
+    # Call z/OS fact gathering with the 'ipl' subset and filter down to
+    # master_catalog facts.
+    # ##########################################################################
+  - name: Collect z/OS facts on master_catalog.
+    zos_gather_facts:
+      gather_subset:
+        - 'ipl'
+      filter:
+        - 'master*'
+
+    # ##########################################################################
+    # Collected facts can be accessed via the ansible_facts dict.
```

Comment:
This is just me being overly particular, some may not know what dict is , so maybe dict (dictionary) :)